### PR TITLE
refactored: fixed some styling and used component-library components

### PR DESCRIPTION
**Pull request by ShekharDhangar**
refactored some styling

### CODE REVIEW

**Review by Hamza-777 — APPROVED** [review]
Looks good

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (ApurvaSawant11) [inline .vscode/settings.json]: You can delete this while creating a PR

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good!

Just one suggestion. Increase the width of the search bar. Customers usually type at least 2-3 words. The current size is small for that.

File: .vscode/settings.json (hunk 1 above)
Comment (ShekharDhangar) [reply .vscode/settings.json]: ok

Comment (ShekharDhangar) [reply .vscode/settings.json]: also made the search bar width change in component-library, it will shortly reflect here after merging that one

**Pull request merged**